independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1187 @@
+            "description": "Specifies which field to use for sorting. Allowed fields are publicName.dutch, publicName.english, iata, country and city.. Use plus or minus sign to indicate ascending or descending order.",
+            "required": false,
+            "type": "string",
+            "default": "+publicName.dutch",
+            "x-ms-summary": "Sort"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DestinationList"
+            }
+          }
+        }
+      }
+    },
+    "/destinations/{iata}": {
+      "get": {
+        "tags": [
+          "destination-controller"
+        ],
+        "summary": "Retrieve Destination by IATA",
+        "x-ms-summary": "Retrieve Destination by IATA",
+        "description": "Retrieves destination based on IATA code",
+        "x-ms-visibility": "important",
+        "operationId": "retrieveDestinationUsingGET",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "app_id",
+            "in": "header",
+            "description": "Your API app id",
+            "required": true,
+            "type": "string",
+            "x-data-threescale-name": "app_ids",
+            "x-ms-summary": "App ID"
+          },
+          {
+            "name": "app_key",
+            "in": "header",
+            "description": "Your API app key",
+            "required": true,
+            "type": "string",
+            "x-data-threescale-name": "app_keys",
+            "x-ms-summary": "App Key"
+          },
+          {
+            "name": "ResourceVersion",
+            "in": "header",
+            "description": "Resource version of the API",
+            "required": true,
+            "default": "v4",
+            "type": "string",
+            "x-ms-summary": "Resource Version"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "description": "Accept header",
+            "required": true,
+            "default": "application/json",
+            "type": "string",
+            "x-ms-visibility": "internal",
+            "x-ms-summary": "Accept"
+          },
+          {
+            "name": "iata",
+            "in": "path",
+            "description": "IATA code",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "IATA Code",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Destination"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RowType": {
+      "properties": {
+        "Position": {
+          "type": "string",
+          "x-ms-summary": "Belts"
```

Comment:
@Miesjh you have two same `x-ms-summary` with `Belts`, please check if it's correct.

Reply:
Fixed